scripts/xtalk_filter_run.py

```diff
@@ -14,25 +14,25 @@
 a = parser.parse_args()
```

Comment:
I think you meant for this to be `ap = `

Reply:
That's correct. Thanks for catching.